contracts/ExampleNFT.cdc

```diff
@@ -304,8 +301,11 @@ access(all) contract ExampleNFT: NonFungibleToken {
 
         // Create a Collection resource and save it to storage
         let collection <- create Collection()
-        let defaultStoragePath = collection.storagePath
-        let defaultPublicPath = collection.publicPath
+
+        let identifier = "cadenceExampleNFTCollection"
```

Comment:
For compatibility with previous paths
```suggestion
        let identifier = "exampleNFTCollection"
```
L232 and L233 need to be updated too
For reference, here's the previous version: https://github.com/onflow/flow-nft/blob/4c0c684baed98e2ffd7a8f9d27973c1db5dd1239/contracts/ExampleNFT.cdc#L390

Reply:
@SupunS Were you going to make this change or should I just do it?

Reply:
Added in https://github.com/onflow/flow-nft/pull/205/commits/692b3dea2d0051b35516c6754ac7cc05de1d434b

Reply:
oh, just realize there are many places that needs updating. @joshuahannan might be better if you can update it. I could break things unknowingly.

Reply:
okay. I'll pull down your branch and make the changes